nexus/src/app/background/sync_switch_configuration.rs

```diff
@@ -587,6 +590,154 @@ impl BackgroundTask for SwitchPortSettingsManager {
                             bgp_announce_prefixes.insert(bgp_config.bgp_announce_set_id, prefixes);
                         }
 
+                        let ttl: Option<u8> = match peer.min_ttl.as_ref() {
```

Comment:
What's the type of `peer.min_ttl`? I thought it was an `Option<u8>` already. If not then I think we should fix that.

Reply:
This nastiness is all gone and the underlying DB type is now an `SqlU8`